tests/CallbackTest.php

```diff
@@ -73,14 +81,25 @@ public function testCallbackTrigger(): void
 
     public function testViewUrlCallback(): void
     {
-        $var = null;
-
+        $cbApp = \Atk4\Ui\Callback::addTo($this->app, ['urlTrigger' => 'aa']);
         $v1 = \Atk4\Ui\View::addTo($this->app);
-        $cb = \Atk4\Ui\Callback::addTo($v1);
+        $cb = \Atk4\Ui\Callback::addTo($v1, ['urlTrigger' => 'bb']);
 
-        // simulate triggering
-        $_GET[Callback::URL_QUERY_TRIGGER_PREFIX . $cb->name] = '1';
+        $this->simulateCallbackTriggering($cbApp);
+        $this->simulateCallbackTriggering($cb);
 
+        $expectedUrlCbApp = '?' . Callback::URL_QUERY_TRIGGER_PREFIX . 'aa=callback&' . Callback::URL_QUERY_TARGET . '=aa';
+        $expectedUrlCb = '?' . /*Callback::URL_QUERY_TRIGGER_PREFIX . 'aa=1&' .*/ Callback::URL_QUERY_TRIGGER_PREFIX . 'bb=callback&' . Callback::URL_QUERY_TARGET . '=bb';
```

Comment:
test of the current/wrong behaviour, will be fixed in https://github.com/atk4/ui/pull/1651